packages/yew/src/callback.rs

```diff
@@ -123,12 +69,12 @@ impl<IN> Default for Callback<IN> {
     }
 }
 
-impl<IN: 'static> Callback<IN> {
+impl<IN: 'static, OUT: 'static> Callback<IN, OUT> {
     /// Changes the input type of the callback to another.
     /// Works like the `map` method but in the opposite direction.
```

Comment:
Maybe we could improve the message to actually state what it does
```suggestion
    /// Creates a new callback from another callback and a function
    /// That when emited will call that function and will emit the original callback
```